cpp/ql/src/Likely Bugs/Underspecified Functions/ImplicitFunctionDeclaration.ql

```diff
@@ -0,0 +1,41 @@
+/**
+ * @name Implicit function declaration
+ * @description An implicitly declared function is assumed to take no
+ * arguments and return an integer. If this assumption does not hold, it
+ * may lead to unpredictable behavior.
+ * @kind problem
+ * @problem.severity warning
+ * @precision medium
```

Comment:
I agree. The sample results we've seen on 88 projects are good enough for me to raise this query to `high`.

Reply:
> I'd like to suggest we consider raising this up to a warning-high, if we can build enough confidence to do so. As a warning-medium it will contribute to the collection of queries that are run but not displayed on LGTM - i.e. that are executed but seen by relatively few users.

Do you think the results justify a warning-high?